coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
+    aspects = defaultdict(lambda: [])
```

Comment:
[From what I understand](http://stackoverflow.com/questions/6589814/what-is-the-difference-between-dict-and-collections-defaultdict) they are same type of things but defaultdict have... default value. CMIIW, in this case if `aspects` is not defined in bear declaration then we can safely call SomeAspectsBear.aspects['detect'] without explicitly handling dict key not found error.